op-batcher/batcher/batch_submitter.go

```diff
@@ -73,7 +73,11 @@ func Main(version string, cliCtx *cli.Context) error {
 				l.Error("error starting metrics server", err)
 			}
 		}()
-		opmetrics.LaunchBalanceMetrics(ctx, l, registry, "", batchSubmitter.L1Client, batchSubmitter.From)
+		l1Client, err := dialEthClientWithTimeout(ctx, cfg.L1EthRpc)
```

Comment:
Why do you need to dial this separately?

Reply:
Very unfortunately because `opmetrics` takes the explicit struct pointer `*ethclient.Client` which I cannot re-use the `batchSubmitter`'s L1Client interface for